src/tools/idlc/idlc.c

```diff
@@ -266,13 +322,19 @@ int32_t idlc_parse(ddsts_type_t **typeptr)
   }
```

Comment:
Some nitpicking: `fclose` (line 321) on `NULL` value results in undefined behaviour

Reply:
fixed